### PR TITLE
fix(app): add reanalyze to overflowMenu on landing page

### PR DESCRIPTION
**Pull request by koji**
# Overview
This PR will add reanalyze to overflowMenu on Protocols landing page. This will fix [RAUT-119](https://opentrons.atlassian.net/jira/software/c/projects/RAUT/boards/75?modal=detail&selectedIssue=RAUT-119)

Screen Shot 2022-08-09 at 1 29 22 PM

In terms of the design, this got confirmation from Emily

# Changelog
- add reanalyze to overflowMenu on protocols landing page
- move `reanalyze` from `protocol-details.json` to `shared.json` and update other overflowMenu i18n
- add a test case for `reanalyze` button in overflowMenu

# Review requests
- [ ] overflowMenu on Protocols landing page has `reanalyze`
- [ ] reanalyze button works as expected

# Risk assessment
- low

### CODE REVIEW

File: app/src/assets/localization/en/shared.json [hunk 1]
```diff
@@ -37,5 +37,6 @@
   "proceed_to_setup": "Proceed to setup",
   "a_software_update_is_available": "A software update is available for this robot. Update to run protocols.",
   "robot_is_busy_no_protocol_run_allowed": "This robot is busy and can’t run this protocol right now. <robotLink>Go to Robot</robotLink>",
+  "reanalyze": "Reanalyze"
```
Comment (brenthagen) [inline app/src/assets/localization/en/shared.json]: needs a comma: `"reanalyze": "Reanalyze",`

Comment (koji) [reply app/src/assets/localization/en/shared.json]: Thanks, I'm not sure how I missed this.

**Comment by koji** [comment]
- [x] fix check-js errors (merge edge)

**Review by brenthagen — APPROVED** [review]

**Comment by koji** [comment]
- [x] resolve conflicts

**Pull request merged**